.drone.star

```diff
@@ -1,6 +1,12 @@
 """oCIS CI definition
 """
 
+# Production release tags
+# NOTE: need to be updated if new production releases are determined
+# - follow semver
+# - omit 'v' prefix
+PRODUCTION_RELEASE_TAGS = ["5.0", "7.0.0"]
```

Comment:
This treats the following tags as production:
- `5.0.x`, `5.0.x-alpha.x`, `5.0.x-beta.x`, `5.0.x-rc.x` (`5.0<anything>`)
- `7.0.0`, `7.0.0-alpha.x`, `7.0.0-beta.x`, `7.0.0-rc.x` (`7.0.0<anything>`)

Reply:
Perfect.